IFC4x3/Properties/f/FiberType_0_QAybmkz5UQ9pCD5EbZDf/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="FiberType_0_QAybmkz5UQ9pCD5EbZDf" Name="FiberType" UniqueId="3e68af25-c2ef-4579-a273-30d14e963369" PrimaryDataType="IfcLabel" />
```

Comment:
Should this be an enumeration? Also renamed with name distinct from property below?

Reply:
I have changed the name to FiberMode and modify the property to PEnum. Commit : 1502e35a1872570498aaf3ca276776848d74b6dd